server-worker/src/main/scala/org/apache/celeborn/service/deploy/worker/Worker.scala

```diff
@@ -44,14 +44,16 @@ import org.apache.celeborn.common.util.{ShutdownHookManager, ThreadUtils, Utils}
 import org.apache.celeborn.server.common.{HttpService, Service}
 import org.apache.celeborn.service.deploy.worker.storage.{PartitionFilesSorter, StorageManager}
 
-private[deploy] class Worker(
+private[celeborn] class Worker(
     override val conf: RssConf,
     val workerArgs: WorkerArguments)
   extends HttpService with Logging {
 
+  @volatile private var stopped = false
```

Comment:
I was thought to name it as `started` but changed to `stopped` eventually because 
the `def initialize()` blocked by `rpcEnv.awaitTermination()`, it's not suitable to wrap by `synchronized { ... }` like `def close()`